scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
Trouble is, there's no straightforward way to get the fully qualified name. I can pull out the logger's name (which is by convention the dotted path of the module) and the function (which i guess comes from the stack frame) but not the class.

I think we're spending too much effort on designing a perfect logging format, when we could work on the scrapers themselves, e.g. getting them into production :)